src/pages/cadastrar-demanda/cadastrar-demanda.css

```
@@ -0,0 +1,115 @@
+
+.container{
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+.content{
+    background-color: #E8ECEF;
+    border-radius: 0;
+    align-self: center;
+    font-weight: 700;
```

Mesmo caso do tamanho da letra
```suggestion
    font-weight: 700;
```